explainaboard/meta_analyses/rank_flipping.py

```diff
@@ -0,0 +1,185 @@
+class RankFlippingMetaAnalysis:  # (can inherit from an abstract MetaAnalysis class)
+    def __init__(self, model1_report, model2_report):
```

Comment:
it would be nice if we could add type info to these arguments, for example, `model1_report:dict`